### DISCUSSION TITLE
MailKit Provider Mail Priority - Low

### DISCUSSION THREAD

**Discussion by james7342** [dnnsoftware/Dnn.Platform#4617] [General] [answered]
Just changed to using MailKit in DNN 9.9.1. Have now noticed all mail messages sent from the site are sent with low priority. Is there a way to set it to normal? If there isn't I can open an enchantment to address. Has anyone else noticed this?![image](https://user-images.githubusercontent.com/27960723/115564301-3472b600-a26d-11eb-9517-fbe0896daf8f.jpeg)

**Comment by bdukes** [comment]
You're right, there's a mismatch here. Here's where `MailKitMailProvider` sets the priority:
https://github.com/dnnsoftware/Dnn.Platform/blob/2e68b5d00bb8d551ef87a557356263edfb6b510a/DNN%20Platform/Library/Services/Mail/MailKitMailProvider.cs#L198

We're converting from [`System.Net.Mail.MailPriority`](https://docs.microsoft.com/en-us/dotnet/api/system.net.mail.mailpriority?view=netframework-4.8) to [`MimeKit.MessagePriority`](http://www.mimekit.com/docs/html/T_MimeKit_MessagePriority.htm) which order their values differently (`Normal`, `Low`, `High` vs. `NonUrgent`, `Normal`, `Urgent`).

Do you want to submit a PR? Otherwise I can get this fix in. Thanks!

**Reply by james7342** [reply]
Not a full fledged developer, more of a code hacker at this point. I'll have to rely on you to assist with this. Plus I don't have a dev environment setup to work on platform changes. Only been playing with module development. 

Thanks for the quick response Brian.

**Comment by bdukes** [comment, marked as answer]
I've submitted #4619 to fix this, thanks for the report!
